src/Components/Members/api.js

```diff
@@ -0,0 +1,24 @@
+export async function fetchMembers() {
+  try {
+    const response = await fetch('http://localhost:4000/api/members');
+    const data = await response.json();
+    return data.data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+
+export async function deleteMember(id) {
+  try {
+    const response = await fetch(`http://localhost:4000/api/members/${id}`, {
```

Comment:
Environment variables should be used here.
process.env.REACT_APP_API_URL + `/members/${id}`

Reply:
Changes made!